apis/clusters/v1beta1/validation.go

```diff
@@ -216,6 +216,30 @@ func validateTwoFactorDelete(new, old []*TwoFactorDelete) error {
 	return nil
 }
 
+func validateIngestNodes(new, old []*OpenSearchIngestNodes) error {
+	if len(old) != len(new) {
+		return models.ErrImmutableIngestNodes
+	}
+
+	if *old[0] != *new[0] {
+		return models.ErrImmutableIngestNodes
+	}
```

Comment:
Do we validate that only 1 or fewer items can be appended to the IngestNodes array while creation?